tests/cpp/predictor/test_gpu_predictor.cu

```diff
@@ -233,6 +231,33 @@ TEST(gpu_predictor, MGPU_Test) {
     delete dmat;
   }
 }
+
+// multi-GPU predictor external memory test
+TEST(gpu_predictor, MGPU_ExternalMemoryTest) {
+  std::unique_ptr<Predictor> gpu_predictor =
+      std::unique_ptr<Predictor>(Predictor::Create("gpu_predictor"));
+  gpu_predictor->Init({std::pair<std::string, std::string>("n_gpus", "-1")}, {});
+
+  gbm::GBTreeModel model = CreateTestModel();
+  const int n_classes = 3;
+  model.param.num_output_group = n_classes;
+  std::vector<std::unique_ptr<DMatrix>> dmats;
+  dmats.push_back(CreateSparsePageDMatrix(9, 64UL));
+  dmats.push_back(CreateSparsePageDMatrix(128, 128UL));
```

Comment:
perhaps, you could add the other dmatrix (that was removed earlier) here now.

Reply:
Done.